devito/arch/archinfo.py

```diff
@@ -14,11 +14,11 @@
 
 __all__ = ['platform_registry', 'get_cpu_info', 'get_gpu_info', 'get_nvidia_cc',
            'Platform', 'Cpu64', 'Intel64', 'Amd', 'Arm', 'Power', 'Device',
-           'NvidiaDevice', 'AmdDevice',
+           'NvidiaDevice', 'AmdDevice','IntelDevice',
```

Comment:
`, 'IntelDevice',`
This whitespace  after comma should fix everything in flake8 
https://github.com/devitocodes/devito/runs/5155237472?check_suite_focus=true